wiki/People/The_Team/pl.md

```diff
@@ -1,50 +1,51 @@
----
-outdated: true
----
-
 Załoga
 ========
 
+Wymienione poniżej osoby są rdzeniem zespołu Osu i głównie odpowiadają za pomoc w sprawianiu, by gra była wspaniała dla wszystkich.
+
 Twórca
 ----------------------
 
 | Pseudonim | Opis |
 | ---- | ----------- |
-| [peppy](https://osu.ppy.sh/u/2) | Prowadzenie projektu |
+| [![][flag_AU] peppy](https://osu.ppy.sh/u/2) | Prowadzenie projektu |
```

Comment:
może *Twórca projektu* ? całe to wyrażenie brzmi dziwnie jak masz rzeczownik od czasownika